bonus_file_with_defs.rb

```diff
@@ -0,0 +1,274 @@
+#################################################################################
+###########################    General methods    ###############################
+#################################################################################
+require 'fileutils'
```

Comment:
We already discussed that name of the file is not ok. But I still see "bonus_file_with_defs.rb"